docs/sources/_index.md

```diff
@@ -0,0 +1,75 @@
+---
+description: The GitHub data source lets you visualize GitHub data in Grafana dashboards.
+keywords:
+  - data source
+  - github
+  - github repository
+  - API
+labels:
+  products:
+    - oss
+    - enterprise
+    - cloud
+menuTitle: GitHub data source
+title: GitHub data source plugin for Grafana
+weight: 10
+---
+
+# GitHub data source plugin for Grafana
+
+The Grafana GitHub data source plugin provides, allow you to track and analyze GitHub data directly within Grafana, providing insights and visualizations for your GitHub repositories and projects.
+
+{{< docs/play title="GitHub data source plugin demo" url="https://play.grafana.org/d/cdgx261sa1ypsa/3-single-repo-with-override-examples" >}}
+
+The plugin provide the features listed below:
+
+## Query types
+
+- Commits
+- Issues
+- Contributors
+- Tags
+- Releases
+- Pull requests
+- Labels
+- Repositories
+- Milestones
+- Packages
+- Vulnerabilities
+- Projects
+- Stargazers
+- Workflows
+- Workflow usage
+
+## Supported features
+
+- Visualize queries
+- Template variables
+- Annotations
+- Query caching
+
+## Caching
+
+Caching on this plugin is always enabled.
+
+## Requirements
+
+The GitHub data source plugin has the following requirements:
+
+- A free [GitHub](https://github.com/) or a [GitHub Enterprise](https://github.com/enterprise) account.
+- Any of the following Grafana flavours:
+  - Grafana OSS
+  - Free or paid [Grafana Cloud](https://grafana.com/pricing/) server
+  - An [activated on-prem Grafana Enterprise](https://grafana.com/docs/grafana/latest/enterprise/license/activate-license/) server.
+  - Port `3000` enabled.
```

Comment:
I'm still not entirely sure what this means? Why does one need port 3000 and how is it "enabled"? I know this is the default port for the Grafana server but what about in Grafana Cloud?

Reply:
This is just a additional information. I referenced some other plugin docs and in the requirements section. The part is not mentioned. So I will simply remove it

Reply:
removed the port part. fixed now.